main.py

```diff
@@ -107,6 +107,79 @@ def get_name_tag(resource_type, name):
 )
 
 
+# Initialize the ELBv2 (ALB) client
+elbv2 = boto3.client('elbv2')
+
+# Create a Security Group for the ALB
+alb_sg = ec2.create_security_group(
+    Description='ALB security group',
+    GroupName='alb-sg',
+    VpcId=vpc.id,
+    TagSpecifications=get_name_tag('security-group', 'boto3-alb-sg')
+)
+
+# Allow inbound HTTP traffic to the ALB Security Group
+alb_sg.authorize_ingress(
+    IpPermissions=[
+        {
+            'IpProtocol': 'tcp',
+            'FromPort': 80,
+            'ToPort': 80,
+            'IpRanges': [{'CidrIp': '0.0.0.0/0'}]
+        }
+    ]
+)
+
+# Create an Application Load Balancer
+alb_response = elbv2.create_load_balancer(
+    Name='boto3-application-load-balancer',
+    Subnets=[subnet.id for subnet in public_subnets],
```

Comment:
The issue is that 2 subnets should be added from different availability zones. Currently the creation of the public subnets is non-deterministic, we should specify the availability zones during creation.

Reply:
Unfortunately issue #20 must be addressed first to proceed with this PR.